web/angular-wallet/src/app/layout/components/burst-fee-selector/burst-fee-selector.component.ts

```diff
@@ -31,7 +31,10 @@ export class BurstFeeSelectorComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.feeNQT = this.convertFeeToBurst(this.fees.standard).toString();
+    // avoids ExpressionChangedAfterItHasBeenCheckedError
+    setTimeout(() => {
```

Comment:
:hankey: smells like teen spirit - but you said why

Reply:
yeah gross, and actually doesnt "break" the app, but does cause the build to fail.